operations/common-operation.go

```diff
@@ -70,6 +79,13 @@ func CreateClientConfig(context *ClientContext) *sarama.Config {
 		config.Net.TLS.Config = tlsConfig
 	}
 
+	if context.Sasl.Enabled {
+		output.Infof("SASL is enabled.")
```

Comment:
the log should be printed with `output.Debugf()` so that it only shows up when `--verbose` flag is set.

Reply:
Ok, that sounds good.